schema/deploy/tables/ciip_user.sql

```diff
@@ -3,83 +3,57 @@
 
 begin;
```

Comment:
This change should be a `..._001` migration instead, as the change is not idempotent (see https://sqitch.org/docs/manual/sqitch-rework/)